ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -0,0 +1,56 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
```

Comment:
this tag image refs an existing image? If yes, should we change `4.14` to something like latest or stable? Do not know where these tags are listed

Reply:
An existing image.
<ci-registry>/ocp/builder:rhel-8-golang-1.20-openshift-4.14

Reply:
from what I searched (https://github.com/openshift-eng/ocp-build-data/blob/openshift-4.14/streams.yml), there is no image name without the ocp version in it. I will just add a note in our docs that this does not have to match the ocp version of the job

resolving